dev_docs/tutorials/adding_performance_metrics.mdx

```diff
@@ -7,76 +7,156 @@ date: 2022-07-07
 tags: ['kibana', 'onboarding', 'setup', 'performance', 'development', 'telemetry']
 ---
 
-Reporting performance events is as easy as shown below. However, it's highly suggested to follow guidelines for what to report in certain fields:
+## Reporting performance events
 
-```typescript
-import { reportPerformanceMetricEvent } from '@kbn/ebt-tools';
+### Simple performance events
+
+Lets assume we intend to report the performance of a specific action called `APP_ACTION`.
+In order to do so, we need to first measure the timing of that action.
+Once we have the time measurment, we can use the `reportPerformanceMetricEvent` API to report it.
+
+The most basic form of reporting would be:
 
+```typescript
 reportPerformanceMetricEvent(analytics, {
-  eventName: KIBANA_LOADED_EVENT,
-  meta: {
-    // fields that are searchable-as-keywords but not aggregateable
-    kibana_version: this.coreContext.env.packageInfo.version,
-    protocol      : window.location.protocol,
-  },
-  duration: timing[LOAD_FIRST_NAV],
-  key1    : LOAD_START,
-  value1  : timing[LOAD_START],
-  key2    : LOAD_BOOTSTRAP_START,
-  value2  : timing[LOAD_BOOTSTRAP_START],
-  key3    : LOAD_CORE_CREATED,
-  value3  : timing[LOAD_CORE_CREATED],
-  key4    : LOAD_SETUP_DONE,
-  value4  : timing[LOAD_SETUP_DONE],
-  key5    : LOAD_START_DONE,
-  value5  : timing[LOAD_START_DONE],
+  eventName: APP_ACTION,     
+  duration,                  // Duration in milliseconds
 });
+```
+
+Once executed, the metric would be delivered to the `stack-telemetry` cluster, along side with the event's context.
+The data is updated periodically, so you might have to wait up to 30 minutes to see your data in the index.
+
+Once indexed, this metric will appear in `ebt-kibana` index. 
+It we also be mapped into an additional index, dedicated to performance metrics.
+We recommend using the performance metrics indices, where you can expect the following structure:
```

Comment:
I was assuming this is going to be default because `ebt-kibana-*-performance-metrics` is the only [index pattern](https://telemetry-v2-staging.elastic.dev/s/kibana-performance/app/management/kibana/indexPatterns) we have and default one, so considering that do we want to mention about this ?